public/enhanced-table-vis-params.html

```diff
@@ -191,7 +191,7 @@
     <div class="euiSpacer euiSpacer--s"></div>
     <div class="form-group">
       <label>Hidden columns</label>
-      <input type="text" ng-model="editorState.params.hiddenColumns" class="form-control" placeholder="0,1,..." />
+      <input type="text" ng-model="editorState.params.hiddenColumns" class="form-control" placeholder="1,2,column_name_X..." />
```

Comment:
Here, can you set `placeholder="0,1,column_label_X,..."`

Reply:
Sure. Will do asap